src/components/services/background/background.js

```diff
@@ -0,0 +1,104 @@
+angular.module("bawApp.components.background", [])
+    .service("bgImage", function() {
+        var curBg = "";
+        return {
+            curBg: curBg
+        };
+    })
+    .component("background",{
+        template: "<div ng-repeat='bg in bgs' ng-if=\"bg.state !== 'finished'\" class='background {{bg.state}}' style='background-image:url({{bg.url}});'></div>",
```

Comment:
Multiline this template with an es6 template string (and then you won't need to escape the double quotes either) . Also do not abbreviate `bg`, `bgs` - I'll stop marking abbreviations but fixes are needed for the whole file.